src/client/eppo-client.ts

```diff
@@ -182,21 +209,42 @@ export default class EppoClient {
    * @param defaultValue default value to return if the subject is not part of the experiment sample
    * @returns a boolean variation value if the subject is part of the experiment sample, otherwise the default value
    */
-  public getBooleanAssignment(
+  public getBooleanAssignment = (
     flagKey: string,
     subjectKey: string,
     subjectAttributes: Record<string, AttributeType>,
     defaultValue: boolean,
-  ): boolean {
-    return (
-      this.getAssignmentVariation(
-        flagKey,
-        subjectKey,
-        subjectAttributes,
-        EppoValue.Bool(defaultValue),
-        VariationType.BOOLEAN,
-      ).boolValue ?? defaultValue
+  ) => this.getBooleanAssignmentDetails(flagKey, subjectKey, subjectAttributes, defaultValue).value;
```

Comment:
what's the motivation for changing these to arrow functions?

Reply:
Just for the one-line return. It was just a preference thing; nothing necessary